app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -743,6 +743,9 @@ private void setupActionBar(final StreamInfo info) {
             public void onItemSelected(final AdapterView<?> parent, final View view,
                                        final int position, final long id) {
                 selectedVideoStreamIndex = position;
+                RelativeLayout r = (RelativeLayout) parent.getChildAt(0);
+                ((TextView) r.getChildAt(1)).setTextColor(getResources().getColor(R.color.white));
+                ((TextView) r.getChildAt(2)).setTextColor(getResources().getColor(R.color.white));
```

Comment:
Isn't there any other way to do this? It seems to be a workaround to me, though if there is no other way then ok, I guess the toolbar inflated layout is always the same.

Reply:
I couldn't find any direct way of styling just the first element of a spinner, so had to use this approach.